theano/gpuarray/cudnn_defs.py

```diff
@@ -145,7 +151,7 @@ def get_definitions(cudnn_version=None):
     if None, return definitions for the  most recent supported cuDNN version.
 
     """
-    if cudnn_version is not None and cudnn_version // 1000 == 5:
-        return CuDNNV51()
+    if cudnn_version is not None and cudnn_version // 1000 == 6:
```

Comment:
You seem to keep cudnn v5 support. I'm fine to keep it or remove it now. If it is easy to keep, why not.

If we keep it, you would need to add this:

elif cudnn_version is not None and cudnn_version // 1000 == 6:
    return CuDNNV51()